CHANGELOG.md

```diff
@@ -18,6 +18,12 @@
 * [Breaking] Rename `yarpc.CreateThriftServiceFunc` to `yarpc.ServiceCreateFunc`
   as it is not thrift-specific.
 * Report version metrics for company-wide version usage information.
+* [Breaking] Rename functions inside `package dig` to match other APIs within UberFx:
+        * `Register` to `TryRegister`
```

Comment:
An easy fix, I can just force push head~3 on this branch and we'll be back to `Must`.

`Try*` came from the parity with the Config module